README.md

```diff
@@ -1,4 +1,8 @@
 Telemetry Services
 ------------------
 
-This repository contains schemas for Mozilla Firefox Telemetry pings.
\ No newline at end of file
+This repository contains schemas for Mozilla Firefox Telemetry pings.
+
+These schemas are used to validate incoming submissions at ingestion time. The [RapidJSON](http://rapidjson.org/md_doc_schema.html) library is used for JSON Schema Validation.
```

Comment:
Do we want to call out that only the RapidJSON patterns are supported on our production systems?